packages/react/src/components/SuiteHeader/IdleLogoutConfirmationModal/IdleLogoutConfirmationModal.jsx

```diff
@@ -113,7 +113,12 @@ const IdleLogoutConfirmationModal = ({
   const [stayLoggedInButtonDisabled, setStayLoggedInButtonDisabled] = useState(false);
   // eslint-disable-next-line consistent-return
   useEffect(() => {
-    if (idleTimeoutData && idleTimeoutData?.timeout > 0 && routes) {
+    if (
```

Comment:
Glad to see if fixed here as well.  I just got another defect where the cookie was being created using the wrong domain, and turns out someone was using this class directly vs SuiteHeader.